rio/src/RIO/Prelude/Exit.hs

```diff
@@ -0,0 +1,15 @@
+module RIO.Prelude.Exit
+  ( module System.Exit
+  , exitSuccess
+  , exitFailure
+  ) where
+
+import           Control.Monad.IO.Class
+import           System.Exit hiding( exitFailure, exitSuccess )
+import qualified System.Exit
+
+exitFailure :: MonadIO m => m a
```

Comment:
Please add Haddock comments that include a message like `Lifted version of "System.Exit.exitFailure"` and a `@since 0.1.9.0` comment.

Reply:
Definitely, thanks!